src/views/darkroom.c

```diff
@@ -2221,6 +2224,21 @@ void init_key_accels(dt_view_t *self)
 
   // fullscreen view
   dt_accel_register_view(self, NC_("accel", "full preview"), GDK_KEY_z, 0);
+
+  // undo/redo
+  dt_accel_register_view(self, NC_("accel", "undo"), GDK_KEY_z, GDK_CONTROL_MASK);
+  dt_accel_register_view(self, NC_("accel", "redo"), GDK_KEY_r, GDK_CONTROL_MASK);
```

Comment:
This is the first time i see ``redo`` to default as ``Ctrl+R``.
I think, in all cases i have seen, it is either ``Ctrl+Y`` or ``Ctrl+Shift+Z``.